.github/workflows/handle.yml

```diff
@@ -109,3 +108,85 @@ jobs:
           else
             echo "::notice title=CLI Request Handle::Skip because tool.yaml is not found"
           fi
+
+      - name: 'IAM Request Comment'
+        if: '${{ always()}}'
```

Comment:
Does it make more sense to have `if: '${{ hashFiles(''iam.yaml'') != '''' }}'` here? Not sure why we want to comment if there is no request.

Reply:
Maybe a notice to people who use the wrong file name by mistake?